src/components/Grades/AnimatedEmoji.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect } from "react";

```

Comment:
Par rapport à la suggestion d'après :

```suggestion
import React, { useEffect, useState } from "react";
```